frontend/src/components/Graph.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { VictoryStack, VictoryAxis, VictoryArea, VictoryTheme, VictoryChart, VictoryLine, LineSegment, VictoryLabel} from 'victory';
+import {
+    getResponseByTractIDAndYear
+} from "../utils/apiWrapper";
+
+const STEPS = 5
+
+class Graph extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            data: [],
+            x_labels: [],
+            y_labels: [],
+            tract_id: this.props.tract_id,
+            year: 2010
+        };
+    }
+
+    async componentDidMount() {
+        const response = await getResponseByTractIDAndYear(this.state.tract_id, this.state.year);
+        if (!response.data.result.response_rates["0"]) {
```

Comment:
could you try using `.response_rates[0]` instead of `.response_rates["0"]` and see if it works?